packages/synapse-react-client/src/lib/style/components/_goals.scss

```diff
@@ -78,6 +79,9 @@
         padding: 10px;
         margin-top: 20px;
         margin-left: 0px;
+        &:hover {
+          color: map.get(SRC.$colors, 'white');
+        }
```

Comment:
Style so that Goals text doesn't change on hover for Desktop or Mobile, as in this [comment](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/177/files#r1182797166).